core/templates/pages/contributor-dashboard-page/opportunities-list/opportunities-list.component.ts

```diff
@@ -26,28 +26,35 @@ import { ExplorationOpportunity } from '../opportunities-list-item/opportunities
 import constants from 'assets/constants';
 import { Subscription } from 'rxjs';
 
+type LoadOpportunities = () => Promise<{
```

Comment:
Interesting fix :) Although you're describing a function, though, the type should still be a *noun* (just like the word "function" is a noun).

Perhaps something like ExplorationOpportunitiesFetcherFunction or similar?

Reply:
Done